ansible/roles/jboss_eap_install/defaults/main.yaml

```diff
@@ -2,3 +2,4 @@ jboss_zip_path: 'jboss-eap-7.0.0.zip'
 jboss_base_dir: '{{jboss_user_dir}}'
 jboss_log_dir: '/var/log/jboss'
 jboss_service_name: 'jboss-eap-7'
+systemd: False
```

Comment:
According to the YAML specification boolean should be either true or false (lower case) http://yaml.org/spec/1.2/spec.html#id2803629